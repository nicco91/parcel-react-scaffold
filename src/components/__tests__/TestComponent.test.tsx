import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import TestComponent from '../TestComponent';

describe('TestComponent', () => {
  it('should render', () => {
    const { getByText } = render(<TestComponent />);
    expect(getByText('TestComponent')).toBeInTheDocument();
  });

  it('should match the snapshot', () => {
    const { container } = render(<TestComponent />);
    expect(container).toMatchSnapshot();
  });
});
