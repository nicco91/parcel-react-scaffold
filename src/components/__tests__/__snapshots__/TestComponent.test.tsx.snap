// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent should match the snapshot 1`] = `
<div>
  <div>
    TestComponent
  </div>
</div>
`;
